schema/omega-services.async.api.yaml

```diff
@@ -414,6 +545,222 @@ components:
           items:
             $ref: '#/components/schemas/GeneralDatedDescription'
 
+    ListAgentParameters:
+      type: object
+      properties:
+        type:
+          $ref: '#/components/schemas/AgentTypes'
+          description: Limits the list to one-or-more types of Agent. If omitted, all types of agents are included in the list.
+        authority-file:
+          $ref: '#/components/schemas/AuthorityFile'
+          description: Limits the list of Agent(s) to only those that are (or are not) also Authority Files. If omitted both authorative and non-authorative Agents are included in the list.
```

Comment:
Am I right in thinking we don't have this Authority File information currently - but this is something we will need in the future?

Reply:
Actually, the opposite! At the moment all of the Persons and Corporate Bodies that we have from ILDB are "Authority Files".